### ISSUE TITLE
Download location

### ISSUE THREAD

**Issue by qukler** [enrico-kaack/markdown-clipper#3]
Do you plan to add option to choose and save download location?

**Comment by jonathan-s** [comment]
Yeah that would be really nice, an option to have a default download location so the only thing you need to do is click the button and no cognitive load on where to save the file.
